modules/swagger-core/src/main/java/io/swagger/core/filter/SpecFilter.java

```diff
@@ -1,11 +1,7 @@
 package io.swagger.core.filter;
 
 import io.swagger.model.ApiDescription;
-import io.swagger.models.Model;
-import io.swagger.models.Operation;
-import io.swagger.models.Path;
-import io.swagger.models.Swagger;
-import io.swagger.models.Tag;
+import io.swagger.models.*;
```

Comment:
In general, we've moved away from wildcard imports.  Not a blocker to this PR of course, just making sure you've seen the [style guides](https://github.com/swagger-api/swagger-core/wiki/Coding-Standards)